### PR TITLE
Upgraded the code in  Script and accomodated the user interactivity

### PR DESCRIPTION
**Pull request by nimeshkverma**
@DrCricket 
Have a look at the changes which I made and understand them
When done notify me so that I can merge the PR.
Update your branch and then test the script accordingly!

### CODE REVIEW

File: mongoschema/mongoschema.py [hunk 1]
```diff
@@ -1,109 +1,174 @@
...
+            Returns PrettyTable object built using the key_type dictionary
+
+            :param key_type_count: The distribution of key types
+            :type key_type_count: dictionary
+
+            :return: PrettyTable built from the key type dict
+            :rtype: PrettyTable object
+
+        """
+        pretty_table_defaul_value = [
```
Comment (utsavtiwary04) [inline mongoschema/mongoschema.py:69]: Values type and Values percentage was needed. +1

File: mongoschema/mongoschema.py [hunk 2]
```diff
@@ -1,109 +1,174 @@
...
+    def get_schema(self):
+        """
+                Returns the schema related stats of a MongoDB collection
+
+                :return: total number of docs and PrettyTable
+                :rtype: int and PrettyTable object
+
+        """
+        total_docs = 0
+        key_type_count_default_value = {
```
Comment (DrCricket) [inline mongoschema/mongoschema.py]: Are we sure there are no other types ?

File: mongoschema/mongoschema.py [hunk 3]
```diff
@@ -1,109 +1,174 @@
...
-
-
 
+class Schema(object):
 
+    "Gets the schema of a MongoDB collection"
+
+    DEFAULT_MONGO_URI = 'mongodb://localhost:27017/'
+
+    def __init__(self, db_name, collection_name, where_dict={}, limit=0, mongo_uri=DEFAULT_MONGO_URI):
```
Comment (utsavtiwary04) [inline mongoschema/mongoschema.py:14]: Port missing ?

File: mongoschema/mongoschema.py [hunk 4]
```diff
@@ -1,109 +1,174 @@
...
+            str: 0,
+            bool: 0,
+            dict: 0,
+            list: 0,
+            set: 0,
+            tuple: 0,
+            None: 0,
+            object: 0,
+            "other": 0,
+            "total": 0
```
Comment (utsavtiwary04) [inline mongoschema/mongoschema.py:113]: Are we sure there are no other types ?

File: mongoschema/mongoschema.py [hunk 5]
```diff
@@ -1,109 +1,174 @@
...
+        """
+        pretty_table_defaul_value = [
+            'Key', 'Occurrence Count', 'Occurrence Percentage', 'Values Type', 'Value Type Percentage']
+        result_table = PrettyTable(pretty_table_defaul_value)
+
+        for key, key_types in key_type_count.iteritems():
+            max_key_type_count = max(key_types.values())
+            max_key_type = [key_type for key_type, key_type_count in key_types.iteritems(
+            ) if key_type_count == max_key_type_count][0]
+
```
Comment (utsavtiwary04) [inline mongoschema/mongoschema.py:77]: Index information ?

**Pull request merged**